packages/nuxt/src/runtime/utils.ts

```diff
@@ -33,10 +34,13 @@ export function extractErrorContext(errorContext: CapturedErrorContext | undefin
  *
  * Exported only for testing
  */
-export function addSentryTracingMetaTags(head: NuxtRenderHTMLContext['head']): void {
+export function addSentryTracingMetaTags(head: NuxtRenderHTMLContext['head'], debug?: boolean): void {
   const metaTags = getTraceMetaTags();
 
   if (metaTags) {
+    if (debug) {
+      logger.log('Adding Sentry tracing meta tags to HTML page:', metaTags);
+    }
```

Comment:
l: I think I caused this confusion in our offline convo: We don't even need the `debug` flag check here because the logger won't emit anything unless people init the SDK with `debug: true`.